services/rsources/handler_test.go

```diff
@@ -1019,3 +1020,7 @@ func getDB(conn string, maxOpenConns int) *sql.DB {
 	Expect(err).NotTo(HaveOccurred())
 	return db
 }
+
+func randomString() string {
+	return strings.ReplaceAll(uuid.Must(uuid.NewV4()).String(), "-", "")
+}
```

Comment:
FYI, we got a helper for random strings, maybe it can work well in this case: https://github.com/rudderlabs/rudder-server/blob/master/testhelper/rand/randString.go